.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+This PR addresses
+https://github.com/covid-projections/covid-data-model/issues/< >
+
+
+Please Check
+- [ ] Are you using data source loaders instead of reading data raw?
```

Comment:
Will everybody working in this codebase know exactly what this means?  (FWIW- I do not.)

Perhaps it could be expanded to something like:
> If you are adding a new data source, did you add a loader for it in `some/repo/path/` rather than reading it raw?